### ISSUE TITLE
Are data trained only on train set, or on both train set and validation set

### ISSUE THREAD

**Issue by MaugrimEP** [openai/guided-diffusion#136]
Hey,
Since there is no need for a validation set, have the pretrained models been trained on the train set only, or on both [train + valid] set?
Best

**Comment by MaugrimEP** [comment]
https://github.com/openai/guided-diffusion/tree/main/datasets
Ok it's only training dataset :>
